### PR TITLE
[RG-459] Initial implementation of IP Dialog box

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
### Motivate of the pull request
 - [ ] To address an existing issue. If so, please provide a link to the issue: <issue id>
 - [x] Breaking new feature. If so, please describe details in the description part.

 ### Describe the technical details
image

 ### Which part of the code base require a change
 
 - [x] Library: ipgenerate, ipconfigurator, foedagcore
 - [ ] Plug-in: <Specify the plugin name>
 - [ ] Engine
 - [ ] Documentation
 - [ ] Regression tests
 - [ ] Continous Integration (CI) scripts

### CODE REVIEW

File: src/IpConfigurator/IPDialogBox.cpp [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+  ui->groupBoxImage->layout()->addWidget(new QLabel{"No image"});
+  ui->paramsBox->setLayout(new QHBoxLayout);
+
+  // Fill and add Parameters box
+  CreateParamFields(true);
+
+  if (!requestedIpName.isEmpty()) handleEditorChanged({}, nullptr);
+  LoadImage();
+
+  setWindowTitle("IP Configuration");
```
Comment (KochynVolodymyr) [inline src/IpConfigurator/IPDialogBox.cpp:147]: @ravic-rs here is the title

Comment (ravic-rs) [reply src/IpConfigurator/IPDialogBox.cpp]: I made the change, thanks!

**Review by ravic-rs — CHANGES_REQUESTED** [review]
Requires error handling check for atleast IP location action. Others seems to be ok.

**Comment by KochynVolodymyr** [comment]
> Requires error handling check for atleast IP location action. Others seems to be ok.

Done

**Review by ravic-rs — APPROVED** [review]

**Pull request merged**